### PR TITLE
Adding Bdbag

### PR DESCRIPTION
**Pull request by bernt-matthias**
Follow up on https://github.com/conda-forge/staged-recipes/pull/9803

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by bernt-matthias** [comment]
ping @saraedum, @sterlingbaldwin and @mikedarcy: According to the guidlines everyone listed as maintainer needs to acknowledge in a comment. Currently mikedarcy and sterlingbaldwin are listed. So please also indicate if you want / want not be included in the maintainer list.

**Comment by mikedarcy** [comment]
I'm fine with being a maintainer. Since we haven't heard back from @sterlingbaldwin during this revived effort I am not sure if we should continue to list him or not. Hopefully he chimes in relatively soon. 

Also, since @bernt-matthias you picked the torch back up on this, do you have any interest in being a maintainer?

**Comment by bernt-matthias** [comment]
> Also, since @bernt-matthias you picked the torch back up on this, do you have any interest in being a maintainer?

Yeah, why not .. then we can share the workload.

File: recipes/bdbag/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    - boto3 >=1.9.5
+    - botocore
+    - awscli
+    - globus-sdk >=1.6.0
+
+test:
+  imports:
+    - bdbag
+  commands:
+    bdbag -h
```
Comment (chrisburr) [inline recipes/bdbag/meta.yaml:42]: I think this is technical valid but it's more conventional to do:
```suggestion
    - bdbag -h
```

File: recipes/bdbag/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  summary: 'The bdbag utilities are a collection of software programs for working with BagIt packages that conform to the BDBag and Bagit/RO profiles.'
+  license: Apache-2.0
+  license_family: Apache
+  license_file: LICENSE
+  description: |
+    The bdbag utilities are a collection of software programs for working with BagIt packages that conform to the BDBag and Bagit/RO profiles.
+
+    The bdbag profiles specify the use of the fetch.txt file, require serialization, and specify what manifests must be provided with a bdbag.
+
+    These utilities combine various other components such as the Bagit-Python bag creation utility and the Bagit-Profiles-Validator utility into a single, easy to use software package.
```
Comment (chrisburr) [inline recipes/bdbag/meta.yaml:55]:
```suggestion
    The bdbag utilities are a collection of software programs for working with
    BagIt packages that conform to the BDBag and Bagit/RO profiles.

    The bdbag profiles specify the use of the fetch.txt file, require
    serialization, and specify what manifests must be provided with a bdbag.

    These utilities combine various other components such as the Bagit-Python
    bag creation utility and the Bagit-Profiles-Validator utility into a single,
    easy to use software package.
```

File: recipes/bdbag/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+test:
+  imports:
+    - bdbag
+  commands:
+    - bdbag -h
+
+about:
+  home: https://github.com/fair-research/bdbag
+  summary: 'The bdbag utilities are a collection of software programs for working with BagIt packages that conform to the BDBag and Bagit/RO profiles.'
```
Comment (chrisburr) [inline recipes/bdbag/meta.yaml:46]: This is too long for the places this is show, can you shorten it?

Comment (bernt-matthias) [reply recipes/bdbag/meta.yaml]: done

**Pull request merged**